hotModuleReplacement.js

```diff
@@ -40,16 +40,26 @@ function updateCss(el, url) {
   if (!url) {
     url = el.href.split('?')[0];
   }
-
+  if (el.isLoaded === false && url == el.href.split('?')[0]) {
```

Comment:
I just change this to `el.isLoaded === false` , and added some test case . I don't know why we need the next condiction `url == el.href.split('?')[0]`?